zipkin4net-owin/Criteo.Profiling.Tracing.Middleware/Criteo.Profiling.Tracing.Middleware.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{6A57C709-4443-4735-94CF-19DE60B00F59}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Criteo.Profiling.Tracing.Middleware</RootNamespace>
```

Comment:
I'm a bit confused about naming here since it's exactly the same as asp.net core middleware one. We definitely need to run a pass on the naming but in the meantime, I think it would be nice to get different names between both.
@crntn What do you think ?